docs/UpgradingToJest28.md

```diff
@@ -7,7 +7,7 @@ Upgrading Jest from v27 to v28? This guide aims to help refactoring your configu
 
 :::info
 
-See [changelog](https://github.com/facebook/jest/blob/main/CHANGELOG.md) for the full list of changes.
+See [changelog](https://github.com/facebook/jest/blob/main/CHANGELOG.md#2800) for the full list of changes.
```

Comment:
this anchor doesn't do anything right now, but will be correct when I make the release